app/src/main/res/values/strings.xml

```diff
@@ -11,13 +11,13 @@
     <string name="play_store">Play Store</string>
 <!--    <string name="google_settings">Google services &amp; preferences</string>-->
     <string name="google_settings">Google settings</string>
-    <string name="component_system_settings">%s system settings</string>
+    <string name="component_system_settings">%s app info</string>
     <string name="gms_network_location_opt_in">Google Location Accuracy</string>
     <string name="no_gms_network_location">
-"Google\'s location service is disabled. Tap \"%1$s\" to learn more.
-If you decide to keep it disabled, consider enabling \"%2$s\" option to improve app compatibility (a lot of apps won\'t function properly when this service is disabled)."
+"Google’s location service is disabled. Tap “%1$s” to learn more.
+If you want to keep it disabled, consider enabling “%2$s” option to improve app compatibility."
```

Comment:
I used "decide" instead of "want" because it's disabled by default, and user is expected to read an explanation in the "Google Location Accuracy" screen and then decide if they want to enable it.
"a lot of apps won\'t function properly when this service is disabled" comment is important, because only apps that request PRIORITY_HIGH_ACCURACY will get reliable access to location when "Google Location Accuracy" is disabled.

Reply:
I think "want" still fits better in that case, as it's a conscious choice by the user.

How about this for clarifying app compatibility issues?

```
Google’s location service is disabled. Tap “%1$s” to learn more.
If you want to keep it disabled, consider enabling “%2$s” to avoid breaking apps.
```